.github/copilot-instructions.md

```diff
@@ -0,0 +1,168 @@
+# Copilot Instructions for ddev-php8.5
```

Comment:
The two ost important things to do here after repo overview are:
* Reference the general instructions in https://github.com/ddev/.github/AGENTS.md
* Allow access to `*.ddev.site` for tests